core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_calendar.scss

```diff
@@ -238,11 +244,12 @@ p-calendar.p-calendar-clearable {
     }
 
     .p-calendar-clear-icon {
-        margin-top: -12px;
-        color: $color-palette-gray-500;
+        margin-top: -7px;
```

Comment:
This value can change due to the font size update [PR](https://github.com/dotCMS/core/pull/29205)

Reply:
According to the storybook, it works fine!

<img width="459" alt="Screenshot 2024-08-16 at 10 11 46 AM" src="https://github.com/user-attachments/assets/d705f31e-4a53-483c-b793-65bfc418e068">